### ISSUE TITLE
Pixi 8 compatibility

### ISSUE THREAD

**Issue by kareljuricka** [pixijs/pixi-projection#118]
Hi, 

is there some plane to make pixi-projection compatible with the pixi 8? Now it's fails during rendering method.

Thank you

**Comment by ivanpopelyshev** [comment]
Yes. I already converted tilemap but people cant help me with publishing examples :)

**Comment by snowcone-team** [comment]
Thoughts on when v8 compatibility will arrive?

**Comment by lionettiluca** [comment]
Hi, any news about this?

**Comment by linh-vanngo-thermo** [comment]
Hi @ivanpopelyshev hope you're doing well...  Any news on this please?

**Comment by ivanpopelyshev** [comment]
> Hi [@ivanpopelyshev](https://github.com/ivanpopelyshev) hope you're doing well... Any news on this please?

Well, I finally have more time, so, quite possible :) But waiting for merge of this https://github.com/pixijs/pixijs/pull/10700  and this https://github.com/pixijs/pixijs/pull/10702

**Comment by hoomanaskari** [comment]
Hi, Any update on this?

**Comment by ivanpopelyshev** [comment]
> Hi, Any update on this?

If you need Quads , there is a official demo in pixi-v8 i think. Something  like that was definitely merged in main pixi

**Comment by hoomanaskari** [comment]
> > Hi, Any update on this?
> 
> If you need Quads , there is a official demo in pixi-v8 i think. Something like that was definitely merged in main pixi

Is there a chance you could share a link to the docs/demos for that, I looked everywhere and could not find any refs to that! Thanks!

**Comment by GoodBoyDigital** [comment]
@hoomanaskari is this what you are after -> https://github.com/pixijs/pixijs/blob/dev/tests/visual/scenes/mesh/perspective-mesh.scene.ts

**Comment by hoomanaskari** [comment]
This would be a very limited version of what I am looking for. 
Can I apply this perspective transform to for example a rounded rectangle or a circle with a video as it's texture?

**Comment by GoodBoyDigital** [comment]
true enough! You can render anything you want to a render texture and use that as the source for the perspective plan. This should get you where you need!

**Comment by hoomanaskari** [comment]
That might work, though it has it's limitations too.

I did not get a chance to test this, or dig deeper into the docs, but is it possible to apply the `applyProjectiveTransformationToPlane` function to graphics, for example to the following:

```javascript
const graphics = new PIXI.Graphics();
// draw a rounded rectangle
graphics.lineStyle(2, 0xff00ff, 1);
graphics.beginFill(0x650a5a, 0.25);
graphics.drawRoundedRect(50, 440, 100, 100, 16);
graphics.endFill();
```

**Comment by yev-stash** [comment]
Hi @ivanpopelyshev, any progress on pixi-projection with pixi8?

**Comment by yev-stash** [comment]
@GoodBoyDigital  hey, do you know if this will ever be compatible with pixi8?

**Comment by ivanpopelyshev** [comment]
I'm looking at all plugins again, maybe this will get updated this month
Pixi now has custom render-plugin support in v8 ,  so pre-reqs are done

**Comment by erdinhrmwn** [comment]
Hello @ivanpopelyshev,

Could you please provide an update on the progress of the Pixi-Projection project using Pixi8?

**Comment by petem24** [comment]
any update?

**Comment by anthony-hyo** [comment]
Hi @ivanpopelyshev, thanks for the updates so far. Is there any rough ETA or progress update regarding PixiJS 8 compatibility?
